### ISSUE TITLE
Cannot tab past widget

### ISSUE THREAD

**Issue by NathanRuth** [AuraQ/AutoCompleteForMendix#60]
I tried the new "Open on focus" which works perfectly, except that when trying to tab to the next item, it actually tabs to the first item on screen instead.

**Comment by ryanb31** [comment]
I have the same problem with tabbing.  In Chrome, it actually works on my machine but in Edge, tabbing out of auto complete seems to go to the first item on the screen.  I updated to the new version which has "Open on Focus" but the tabbing issue still is the same.

Also, Open On Focus does not work.  It did open about once in 12 tries.

**Comment by HanzOhlo** [comment]
Yes, tabbing out of auto-complete is broken after Chrome update (x64) 87.0.4280.66 a few weeks ago (and after Edge browser (chromium) update around the same time).  Tabbing out of auto-complete goes to first item on the screen instead of next item.
